php/libraries/NDB_Menu_Filter.class.inc

```diff
@@ -668,6 +671,7 @@ class NDB_Menu_Filter extends NDB_Menu
         $x = 0;
         foreach ($this->list as $item) {
             //count column
+            // FIXME: Someone should investigate if this can use setTemplateVar
             $this->tpl_data['items'][$x][0]['value'] = $x + $count;
```

Comment:
I don't really like the loop or design of the code as it's currently written, but I don't want to change it in this PR because I don't think it's a place for major refactoring, so I only did the really simple cases and added the note.

This will need to be refactored in order to make the tpl_data variable private for (hopefully?) 18.0, though, otherwise every menu will break.